server/service/src/programs/program_enrolment/program_enrolment_updated.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) fn program_enrolment_updated(
         name: document.name.clone(),
         patient_id: patient_id.to_string(),
         enrolment_datetime,
-        program_patient_id: program.enrolment_patient_id,
+        program_patient_id: program.program_enrolment_id,
```

Comment:
Could you please rename the DB row as well? (just edit the initial migration)

Reply:
And probably filters and sort (if there are any)...

Reply:
[59b9e0f](https://github.com/openmsupply/open-msupply/pull/1215/commits/59b9e0feb385f7e93bd9506fbf5093b1264f75d8)